roles/slurm-client/templates/nhc.conf

```diff
@@ -62,20 +73,42 @@
 # All nodes should have their root filesystem mounted read/write.
    * || check_fs_mount_rw -f /
 
+# All nodes should have their /local filesystem mounted read/write.
+   * || check_fs_mount_rw -f /local
+
 # Controlling TTYs are a good thing!
    * || check_fs_mount_rw -t devpts -s '/(none|devpts)/' -f /dev/pts
 
 # Make sure the root filesystem doesn't get too full.
-   * || check_fs_free / 3%
+   * || check_fs_free / 10%
 
-# Free inodes are also important.
+# Make sure the root filesystem has enough free inodes.
    * || check_fs_ifree / 1k
 
+# Make sure the /var volume doesn't get too full.
+   * || check_fs_free /var 5%
+
+# Make sure the /var filesystem has enough free inodes.
+   * || check_fs_ifree /var 1k
+
 # The following illustrates how to assert an NFSv3 mount (or any other specific mount option).
 #   * || check_fs_mount -s bluearc0:/home -t nfs -o '/(^|,)vers=3(,|$)/' -f /home
-* || check_fs_mount -s gcc-storage001.stor.hpc.local:/ifs/rekencluster/umcgst10/home  -t nfs -o '/(^|,)vers=4(,|$)/' -f /home
-* || check_fs_mount -s gcc-storage001.stor.hpc.local:/ifs/rekencluster/tmp01 -t nfs -o '/(^|,)vers=4(,|$)/' -f /mnt/tmp01
-* || check_fs_mount -s gcc-storage001.stor.hpc.local:/ifs/rekencluster/umcgst10/.envsync/tmp01 /apps -t nfs -o '/(^|,)vers=4(,|$)/' -f /apps
+#* || check_fs_mount -s gcc-storage001.stor.hpc.local:/ifs/rekencluster/umcgst10/home  -t nfs -o '/(^|,)vers=4(,|$)/' -f /home
```

Comment:
See previous comment about comments.

Reply:
Same; requires testing. Hope I can do that today...